postgres-appliance/scripts/configure_spilo.py

```diff
@@ -88,6 +88,7 @@ def write_certificates(environment, overwrite):
 
     ssl_keys = ['SSL_CERTIFICATE', 'SSL_PRIVATE_KEY']
     if set(ssl_keys) <= set(environment):
+        logging.info('Generating custom ssl certificate')
```

Comment:
We are not generating anything here, only writing to files.

Reply:
Is it ok for you if I fix this with "Writing custom SSL certificate"